app/controllers/declaration/RoutingCountriesSummaryController.scala

```diff
@@ -70,12 +70,12 @@ class RoutingCountriesSummaryController @Inject()(
       )
   }
 
+  private def isModeErrorFix(mode: Mode): Boolean = mode == ErrorFix
```

Comment:
Could you not add a method (isErrorFix) to Mode?

Reply:
yeah, I thought about it. I'll do it

Reply:
Fixed